models/breast_density_classification/scripts/createList.py

```diff
@@ -0,0 +1,40 @@
+import json
+
+
+class CreateImageLabelList:
+    def __init__(self, filename):
+        self.filename = filename
+        fid = open(self.filename, "r")
+        self.json_dict = json.load(fid)
+
+    def create_dataset(self, grp):
+        image_list = []
+        label_list = []
+        image_label_list = self.json_dict[grp]
+        for _ in image_label_list:
+            image_list.append(_["image"])
+            label_list.append(_["label"])
+        return image_list, label_list
+
+
+def create_sample_imagelist(json_filename):
+    fid = open(json_filename, "r")
+    file_dict = json.load(fid)
+    train_data = file_dict["Train"][0:40]
+    validation_data = file_dict["Validation"][0:20]
+    test_data = file_dict["Test"][0:20]
+
+    data_dict = {}
+    data_dict["Train"] = train_data
+    data_dict["Validation"] = validation_data
+    data_dict["Test"] = test_data
+
+
+def main():
+    json_filelist = "/raid/Data/MayoClinicData/BreastDensity/patient_img_label_FFDM.json"
```

Comment:
Hi @vikashg , for this file, can I ask if the original dataset is publicly accessible? If it is only for internal use, maybe this file is not needed.

Reply:
Sorry that was the trace left from my testing and I am incorporating all your feed back now and I will soon update. 
Thanks